cmd/tiller-proxy/Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM alpine:3.6
+FROM alpine:3.6 as builder
 RUN apk --no-cache add ca-certificates
+
+FROM scratch
+COPY --from=builder /etc/ssl/certs /etc/ssl/certs
```

Comment:
should we just copy the ca-certificates.crt bundle like in https://github.com/kubeapps/kubeapps/blob/master/cmd/apprepository-controller/Dockerfile#L7? Just want to be consistent, if there is good reason to copy the whole dir we can update the other ones.

Reply:
I didn't know that the rest of certificates there wasn't necessary. It works without them.